wordpress-mobile/WordPress-iOS
# Dashboard Cards: Display latest pages in Pages card

Closes #20440
Design ref: YSMw2nbFAgPpjZLLbYvTfT-fi-96_3834

## Description
Displays the 3 most recently modified pages in the Pages card.

## Screenshot

<img src="https://user-images.githubusercontent.com/25306722/231611063-d7162eb4-9d09-4c2f-bcc1-4c55d10def8a.png" width=200>

## Known Issues
- There is an issue where the date modified is not set correctly when scheduling a page from the app. This can affect the order of the pages, causing the scheduling page to always appear at the top. This will be handled as part of #20521 
- Dark mode is not supported yet. This will be handled as part of #20523
- Sites with no pages are not properly supported yet. This will be handled in #20441

## Testing Instructions

1. Run the Jetpack app
2. Enable the Pages Card Remote Feature Flag from the debug menu
3. Navigate to the dashboard
4. Make sure that the Pages card displays your site's pages
5. Make sure that all page types are displayed (Published, Scheduled, Draft)
6. Make sure that the pages are ordered by the last modified date.
7. Make sure that the page cell matches the design
8. Tap on the floating button
9. Tap Site Page 
10. Fill in the title and content
11. Publish the page
12. Make sure the new page is added to the Pages card

## Regression Notes
1. Potential unintended areas of impact
Dashboard posts card and posts syncing.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing & `DashboardPostsSyncManagerTests`

3. What automated tests I added (or what prevented me from doing so)
`DashboardPostsSyncManagerTests.testSuccessfulPagesSync`

PR submission checklist:
